editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1856,7 +1856,14 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 					if (after != EditorPlugin::AFTER_GUI_INPUT_CUSTOM) {
 						//clicking is always deferred to either move or release
-						clicked = _select_ray(b->get_position());
+
+						//single item selection
```

Comment:
```suggestion
						// Single item selection.
```
Also the above comment could get removed probably. Not sure what it refers to and with your changes there is awkward blank space that makes it disconnected.